crates/rpc/rpc-types/src/eth/block.rs

```diff
@@ -1,5 +1,8 @@
 use crate::Transaction;
-use reth_primitives::{Address, Bloom, Bytes, H256, H64, U256};
+use reth_primitives::{
+    Address, Block as RethBlock, Bloom, Bytes, Header as RethHeader, H256, H64, U256,
```

Comment:
hmm, not sure about the `Reth` prefix because this is not specific to reth itself.
I'd suggest `Primitive` or `Eth` as prefix instead.

Reply:
changing to `Primitive` for all aliases made it much more explicit, hopefully more understandable